src/internal.c

```diff
@@ -26939,8 +26939,13 @@ static int ParseCipherList(Suites* suites,
                 break;
             }
         }
+
+        if (!next) {
+            break;
+        }
+        next++; /* ++ needed to skip ':' */
     }
-    while (next++); /* ++ needed to skip ':' */
+    while (TRUE);
```

Comment:
TRUE is not portable. Please use: `while (1)`

Reply:
The use case for me is to use wolfssl as a ssl backend for libcurl so I can enable HTTP/3 support.

I don't have any other PRs planned except for this one, if that changes I'm send the email as suggested.

Reply:
Hi @alexsn , thank you for those details. Please go ahead and send an email to support so we can figure it out.

Reply:
I don't follow, send an email saying what? Please review this PR request on github?